.dockerignore

```diff
@@ -7,3 +7,6 @@ bin/release
 circle.yml
 pkg
 tags
+*.swp
```

Comment:
That's a limitation I'm willing to accept hahaha

If you're concerned, we could match `.*.sw*`